src/Models/ProjectModel.php

```diff
@@ -27,13 +27,28 @@ public function getProjects(): array
         return $query->fetchAll();
     }
 
-    public function getById(int $id): ProjectEntity
+    //Not sure about this
+//    public function getTask(): array|false
+//    {
+//        $query = $this->db->prepare(
+//     'SELECT `tasks`.`name` AS "taskname",
+//`tasks`.`estimate`,
+//`users`.`name` as "username",
+// `users`.`avatar` AS "icon"
+// FROM `tasks` INNER JOIN `users`
+// ON `tasks`.`user_id` = `users`.`id`;');
+//        $query->execute([]);
+//        return $query->fetch();
+//    }
+
+    public function getProjectName(int $id): array
```

Comment:
This query name is pretty misleading - sounds like it just gets the name of a project, but actually it gets all details about a project. Suggest calling it getById instead